src/code/z_camera_data.c

```diff
@@ -59,8 +59,15 @@ char sCameraModeNames[][12] = {
     "HANG       ", "HANGZ      ", "FREEFALL   ", "CHARGE     ", "STILL      ", "PUSHPULL   ", "BOOKEEPON  ",
 };
 
+/*=====================================================================
+ *                   GENERAL NORMAL0 CAMERA DATA
+ *=====================================================================
+ */
+
+/**
+ * Set: NORMAL0    Mode: NORMAL    Func: NORM1
```

Comment:
I really don't like the `set` abbreviation for setting but I guess for enums it's fine (lack of a better abbreviation), however I think we can afford the full word in comments? (same with Func)